Documentation/gettingstarted/aws.rst

```diff
@@ -116,16 +122,50 @@ To list all of the available AWS instances, the following command can be used:
 
    $ kubectl  -n kube-system exec -ti testing-aws-pod -- aws ec2 describe-instances
 
-It is important for this demo that ``kube-dns`` is working correctly. To know the
-status of ``kube-dns`` you can run the following command:
+Once the secret has been created and validated, the cilium-operator pod must be
+restarted in order to pick up the credentials in the secret.
+To do this, identify and delete the existing cilium-operator pod, which will
+recreated automatically:
+
+.. parsed-literal::
+
+    $ kubectl get pods -l name=cilium-operator -n kube-system
+    NAME                              READY   STATUS    RESTARTS   AGE
+    cilium-operator-7c9d69f7c-97vqx   1/1     Running   0          36h
+
+    $ kubectl delete pod cilium-operator-7c9d69f7c-97vqx
+
+
+
+It is important for this demo that ``coredns`` is working correctly. To know the
+status of ``coredns`` you can run the following command:
 ::
 
     $ kubectl get deployment kube-dns -n kube-system
     NAME       DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
-    kube-dns   1         1         1            1           13h
+    coredns    2         2         2            2           13h
 
 Where at least one pod should be available.
 
+Configure AWS Security Groups
+=============================
+
+Cilium's AWS Metadata filtering capability enables explicit whitelisting
+of communication between a subset of pods (identified by Kubernetes labels)
+with a set of destination EC2 VMs (identified by membership in an AWS security group).
+
+In this example, the destination EC2 VMs are a member of a single AWS security
+group ('sg-0f2146100a88d03c3') and pods with label class=xwing should
+only be able to make connections outside the cluster to the destination
+VMs in that security group.
+
+To enable this, the VMs acting as Kubernetes worker nodes must be able to
+send traffic to the destination VMs that are being accessed by pods.  One approach
+for achieving this is to put all Kubernetes worker VMs in a single 'k8s-worker'
+security group, and then ensure that any security group that is referenced in a
+Cilium toGroups policy has an allow all ingress rule (all ports) for connections from the
+'k8s-worker' security group.    Cilium filtering will then ensure that the only pods allowed
```

Comment:
Will multiple spaces before "Cilium" render? If yes, I think they should be deleted.